man/adoc/bpftrace.adoc

```diff
@@ -188,6 +188,22 @@ Print bpftrace version information.
 Enable verbose messages.
 For more details see the <<Verbose Output>> section.
 
+=== Program Options
+
+You can also pass custom options to a bpftrace program/script itself via positional or named parameters.
+All positional and named parameters should come after the script filename or a double dash if running an inline script with `-e` e.g.
+```
+# bpftrace -e 'BEGIN { ... }' -- p1 --aa=20 --bb p2
+```
+or
+```
+# bpftrace myscript.bt p1 --aa=20 --bb p2
+```
+
+In these examples there are two positional parameters (`p1`, `p2`) and two named parameters (`aa`, which is set to `20`, and `bb`, which is set to `true`).
+Named program parameters require the `=` unless they are boolean parameters (like 'bb' above).
```

Comment:
This (requiring the `=`) is quite an unexpected requirement. Can we lift it?

Reply:
Right, I see why it would be difficult to get rid of the `=`. Yet, I expect users to try to use `--foo hello` as a way to pass the `hello` value to `--foo` (as that's a standard way to do so), so we should think about making their lives easier (with at least some smart warning).

I'll need to think about this more, we can discuss on the office hours.